lib/ansible/executor/module_common.py

```diff
@@ -1000,6 +1000,17 @@ def _find_module_utils(module_name, b_module_data, module_path, module_args, tas
     output = BytesIO()
     py_module_names = set()
 
+    try:
```

Comment:
Only thing that makes me nervous about moving this up is the 3rd case on the original conditional it was contained in (the jsonargs support). I'm not sure how much that's used anymore or if there's really even test coverage, but so long as `_get_ansible_module_fqn` can only fail with ValueError on something wacky related to the jsonargs case, it's probably fine (since that case doesn't care about these values anyway).

Reply:
When looking at the function it only explicitly raises ValueError. It does have some regex function calls and some minor path calls but I don't see why that would raise another type of error.

Reply:
Yep, mostly just meaning so long as it *stays* that way (or that there's not something we both didn't see).